### DISCUSSION TITLE
Wrapper icon not refreshing

### DISCUSSION THREAD

**Discussion by subdesign** [The-Wineskin-Project/WineskinServer#254] [Q&A] [answered]
First of all it's not a bug I'm just curious what is the reason. So when I create a new app with Wineskin, and change its icon, I see the new icon in the Finder or Commander One, but when I click on Launchpad, or start the app and it sits on the Desk, the icon is the original Wineskin one. 
But some days (or weeks) later the icon will refresh in the Launcher/Dock too. Sometimes sooner, sometimes later.
Why the delay? Should I force somehow the wrapper to refresh the icon everywhere?

Thanks for the explanation
A happy Wineskin user!

**Comment by emendelson** [comment, marked as answer]
Two methods that I've used: 

Select the icon and type Cmd-I to open the Get Info pane. The correct icon displays at the top of the Get Info pane, and should then appear on the app itself.

Drag the app to another folder, then back again. This often works also.

**Reply by subdesign** [reply]
Strange. On Get Info pane I see the new icon, so doesn't help, but moving back and forth between dirs the app has now the good icon in Finder...
...But remains the default in Launchpad.
Whatever, thank you :)
